README.md

```diff
@@ -35,13 +46,13 @@ To work with `Agda-LBFT`, you need to have Agda and its standard library install
 
 Once you have installed the correct version of Agda, you should be able to run `./Scripts/run-everything.sh yes` from the root directory of the project and observe successful completion with no errors.
 
-To explore the repo, we suggest starting with the [`LibraBFT.Abstract.Properties`](LibraBFT/Abstract/Properties.agda) module if you are interested in exploring the abstract correctness proof, and with the [`LibraBFT.Yasm.Properties`](LibraBFT/Yasm/Properties.agda) module if you are interested in the system model and its properties.
+To explore the repo, we suggest starting with the [`LibraBFT.Abstract.Properties`](src/LibraBFT/Abstract/Properties.agda) module if you are interested in exploring the abstract correctness proof, and with the [`LibraBFT.Yasm.Properties`](src/LibraBFT/Yasm/Properties.agda) module if you are interested in the system model and its properties.
 
 If you would like to consider contributing to the project, please see our [Contribution Guide](CONTRIBUTING.md).
 
 ## Get Support
 
-* Open a [GitHub issue](https://github.com/oracle/bft-consensus-agda/issues) to provide feedback, raise issues and ask questions.
+* Open a [GitHub issue](../../issues) to provide feedback, raise issues and ask questions.
```

Comment:
GOOD CATCH!

Reply:
BUT WAIT - this PR is for main - not for the blind repo.   It should not change this.   That should only be done in the FM brannch.